core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlCoreEnvironment.kt

```diff
@@ -37,6 +39,11 @@ private object ApplicationEnvironment {
 
   val coreApplicationEnvironment: CoreApplicationEnvironment by lazy(mode = LazyThreadSafetyMode.SYNCHRONIZED) {
     CoreApplicationEnvironment(Disposer.newDisposable()).apply {
+      Logger.setFactory { object : DefaultLogger("") {
```

Comment:
This still creates a new instance each time. Needs moved to a val or object outside the logger factory closure.

Reply:
ahhhh i see. yep makes sense